pubsub/azuresb/azuresb.go

```diff
@@ -26,6 +26,16 @@
 // see URLOpener.
 // See https://godoc.org/gocloud.dev#hdr-URLs for background information.
 //
+// Message Delivery Semantics
+//
+// Azure ServiceBus supports at-least-once semantics in the default Peek-Lock
+// mode; applications must call Message.Ack/Nack after processing a message, or
+// it will be redelivered. However, it also supports a Receive-Delete mode,
+// which essentially auto-acks a message when it is delivered, resulting in
+// at-most-once semantics. See SubscriberOptions.AckFuncForReceiveAndDelete.
```

Comment:
Instead of just "See SubscriberOptions....", say something like "Use ... to choose between the two."

Reply:
Done.